### ISSUE TITLE
Provision CCv2 (Model T) environment for Spartacus

### ISSUE THREAD

**Issue by hackergil** [SAP/cloud-commerce-spartacus-storefront#271]
Provision our own CCv@2(Model T) environment to run spa e2e tests

**Comment by Xymmer** [comment]
This was for getting our tests working in our own modelt
lots of issues getting it working
Gil was waiting for 1811 is out

**Comment by Xymmer** [comment]
We're agreeing to put this on hold because we want to pursue the alternative route.

**Comment by hackergil** [comment]
Blocked by #385

**Comment by hackergil** [comment]
Pull request #519
